Biruk.interlink articles with comments

BrookZewdu opened the pull request:
Modeled the relationships between articles and comments. Also commented the _getAllComments_ feature for review. It will be deleted if it all looks good.

RiyadHassen commented on starter-project-backend-g31/src/routes/comment.ts:
```diff
@@ -2,15 +2,14 @@ import express from 'express';
 const router = express.Router();
 
-router.get('/', getAllComments);
+// router.get('/', getAllComments);
```
would be better to remove this line since we don't use it.

RiyadHassen commented on starter-project-backend-g31/src/tests/__tests__/comments.test.ts:
```diff
@@ -53,15 +53,15 @@ afterEach(async()=>{
 describe('GET: "/api/comment" get the comment route', () => {
```
remove describe block as a whole, comments should be descriptions for a block

RiyadHassen commented on starter-project-backend-g31/src/controllers/comment.ts:
```diff
@@ -3,14 +3,31 @@ import mongoose from 'mongoose';
-    try{
-        const comment = await Comment.find();
-        res.status(200).json(comment);
+// export async function getAllComments(req: Request, res: Response){
```
remove commented section since we don't need this section

BrookZewdu commented:
Hey Riyad, addressed the comments and updated the branch

On Thu, Aug 4, 2022 at 10:42 PM RiyadHassen ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In starter-project-backend-g31/src/routes/comment.ts
> <https://github.com/RealEskalate/Backend-Practice/pull/65#discussion_r938181214>
> :
>
> >          getCommentById,
>          addComment,
>          deleteCommentById,
>          updateCommentById} from  '../controllers/comment';
>
>  const router = express.Router();
>
> -router.get('/', getAllComments);
> +// router.get('/', getAllComments);
>
> would be better to remove this line since we don't use it.
> ------------------------------
>
> In starter-project-backend-g31/src/tests/__tests__/comments.test.ts
> <https://github.com/RealEskalate/Backend-Practice/pull/65#discussion_r938184276>
> :
>
> > @@ -53,15 +53,15 @@ afterEach(async()=>{
>  describe('GET: "/api/comment" get the comment route', () => {
>
> remove describe block as a whole, comments should be descriptions for a
> block
> ------------------------------
>
> In starter-project-backend-g31/src/controllers/comment.ts
> <https://github.com/RealEskalate/Backend-Practice/pull/65#discussion_r938186027>
> :
>
> > @@ -3,14 +3,31 @@ import mongoose from 'mongoose';
>  import {Comment} from'../models/comment';
>
>
> -export async function getAllComments(req: Request, res: Response){
> -    try{
> -        const comment = await Comment.find();
> -        res.status(200).json(comment);
> +// export async function getAllComments(req: Request, res: Response){
>
> remove commented section since we don't need this section
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/RealEskalate/Backend-Practice/pull/65#pullrequestreview-1062495676>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVAAAOHTPDOYPBA3JCZ57ALVXQMK7ANCNFSM55RV3CDQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

RiyadHassen reviewed (approved):
LGTM
